lib/src/components/action_button_component.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+/// [ActionButton] uses for actions
+class ActionButtonComponent extends StatelessWidget {
```

Comment:
Not sure about the `***Component` naming convention here. From my perspective -- it doesn't give any new meaning to the name, `ActionButton` == `ActionButtonComponent`.

Or is there some reason for it?

Reply:
Sometimes we can have a problem when we have CheckBox, but we wan't create custom checkbox, so i prefer use **Component naming cinvention to prevent this problem.

Reply:
Not sure about this approach. For me, It would cause some confusion for me to see `Card` and `CardComponent` used alongside.

I think we can come up with a less abstract name for our custom Widgets. 
This one could be called `HelpOptionButton`, or `HelpActionListEntry` -- since the usage is pretty specific.

Reply:
Yeah, you right, seems prety logic.